addons/website_sale/templates/product_page_templates.xml

```diff
@@ -616,17 +588,20 @@
                 id="add_to_cart_wrap"
                 t-attf-class="{{'d-none' if combination_info['prevent_zero_price_sale'] else 'd-flex'}} flex-grow-1 flex-lg-grow-0 flex-wrap align-items-center gap-2 {{'' if is_50_pc else ' w-100'}}"
             >
-                <a
-                    id="add_to_cart"
-                    role="button"
-                    href="#"
-                    t-attf-class="btn btn-primary js_check_product a-submit flex-grow-1"
-                    t-att-data-show-quantity="is_view_active('website_sale.product_quantity')"
+                <button
+                    name="add_to_cart"
+                    t-attf-class="btn btn-primary flex-grow-1"
+                    data-action="add_to_cart"
```

Comment:
maybe comment to say it's the default but kept to distinguish from the buy_now one for xpaths? (Unless I misunderstood the need of it)